src/couch/src/couch_db.erl

```diff
@@ -796,6 +807,10 @@ set_security(_, _) ->
 set_user_ctx(#db{} = Db, UserCtx) ->
     {ok, Db#db{user_ctx = UserCtx}}.
 
+set_drop_seq(#db{main_pid = Pid} = Db, UuidPrefix, DropSeq) ->
+    check_is_admin(Db),
```

Comment:
does this inherit its user context from the _calculate_drop_seq HTTP API call? Do we need to adjust this if get here via ken?